llama_stack/apis/memory_banks/memory_banks.py

```diff
@@ -89,6 +89,7 @@ class VectorMemoryBank(MemoryBankResourceMixin):
     memory_bank_type: Literal[MemoryBankType.vector.value] = MemoryBankType.vector.value
     embedding_model: str
     chunk_size_in_tokens: int
+    embedding_dimension: Optional[int] = 384  # default to minilm-l6-v2
```

Comment:
we had decided to move this to model metadata. when you register the model you should tell us what its associated dimension is so you don't need to keep specifying it again and again

Reply:
yes, this is still the case. This field is actually inside the instantiated object that is consustructed for a memory bank. The user when they register still just specify the model. We then query the registry to get the model metadata and then populate this field. This is useful since we dont need to keep querying the model for this field. Else every memory provider has to query the registry to get the dimension.

Reply:
And the default value is actually for backcompat reasons.

Reply:
Hm, every memory provider _should_ query the registry for this purpose in my opinion. If not, now we have two sources of truth for this information.

Reply:
hm, maybe this is alright for now.